### PR TITLE
PLANET-7193: Clean up temporal css variables from code

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: [PLANET-7193](https://jira.greenpeace.org/browse/PLANET-7193)

For more information see the description of [#2124](https://github.com/greenpeace/planet4-master-theme/pull/2124)

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [phobos](https://www-dev.greenpeace.org/test-phobos) | [admin](https://www-dev.greenpeace.org/test-phobos/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-phobos/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-phobos) | [composer-local.json](https://github.com/greenpeace/planet4-test-phobos/blob/main/composer-local.json)

:watch: 2023.10.13 13:47:38

File: assets/src/styles/blocks/Accordion/AccordionStyle.scss [hunk 1]
```diff
@@ -5,18 +5,25 @@
...
   _-- {
-    font-family: var(--headings--font-family);
+    font-family: var(--font-family-primary);
   }
 
   &.is-style-light .accordion-content .accordion-headline {
-    --accordion-block-light-style-- {
-      border-color: $grey-20;
-      background: $white;
-      color: var(--body--color);
```
Comment (mleray) [inline assets/src/styles/blocks/Accordion/AccordionStyle.scss:15]: I think this line should stay, right now the font color is white in this case and it's hard to read

File: assets/src/styles/blocks/Articles/ArticlesEditorStyle.scss [hunk 2]
```diff
@@ -21,9 +21,9 @@
     &:hover,
     &:focus,
     &:active {
-      background-color: transparentize($white, .7);
-      border-color: $dark-blue;
-      color: $dark-blue;
+      background-color: rgba($white, .7);
```
Comment (mleray) [inline assets/src/styles/blocks/Articles/ArticlesEditorStyle.scss:24]: Don't we want to replace `$white` with `var(--white)` here?

File: assets/src/styles/blocks/ENForm/components/_enform-full-width-bg.scss [hunk 3]
```diff
@@ -1,5 +1,7 @@
+@import "../../master-theme/assets/src/scss/base/tokens";
+
 .enform-full-width-bg {
-  @include background-before-opacity($light-blue-bg);
+  @include background-before-opacity($beige-100);
```
Comment (mleray) [inline assets/src/styles/blocks/ENForm/components/_enform-full-width-bg.scss:4]: Why don't we use `var(--beige-100)` here? 🤔

File: assets/src/styles/blocks/Happypoint.scss [hunk 4]
```diff
@@ -1,5 +1,7 @@
+@import "../master-theme/assets/src/scss/base/tokens";
+
 .happy-point-block-wrap {
-  @include background-before-opacity($light-blue-bg);
+  @include background-before-opacity($beige-100);
```
Comment (mleray) [inline assets/src/styles/blocks/Happypoint.scss:4]: Same comment here as earlier, why don't we use `var(--beige-100)`?

File: assets/src/styles/blocks/Submenu.scss [hunk 5]
```diff
@@ -33,32 +29,26 @@ div[data-render="planet4-blocks/submenu"] {
...
     }
 
     li {
       &.list-style-bullet {
-        --submenu-block-bullet-item-- {
-          margin-inline-end: 20px;
-          margin-inline-start: 0;
-        }
+        margin-inline-end: 0;
+        margin-inline-start: 16px;
```
Comment (mleray) [inline assets/src/styles/blocks/Submenu.scss:46]: Not a big deal, but here we could use our spacing variables  such as `$sp-2`

**Review by mleray — COMMENTED** [review]
This is something that is also an issue before these changes, but I just noticed that the Table block's caption has the wrong font:

Screenshot 2023-10-19 at 10 18 08

File: assets/src/styles/blocks/Articles/ArticlesEditorStyle.scss (hunk 2 above)
Comment (GP-Dan-Tovbein) [reply assets/src/styles/blocks/Articles/ArticlesEditorStyle.scss:24]: Nope! We've replaced the `transparentize` (SASS function) to `rgba` and to use native implementations. `rgba` does not work with `CSS Variables`, only with `hexa` colors. [That's why we also added `SASS Variables` within the tokens](https://github.com/greenpeace/planet4-design-tokens/pull/8).

File: assets/src/styles/blocks/ENForm/components/_enform-full-width-bg.scss (hunk 3 above)
Comment (GP-Dan-Tovbein) [reply assets/src/styles/blocks/ENForm/components/_enform-full-width-bg.scss:4]: I think you should pass SASS vars instead of CSS vars

**Review by mleray — COMMENTED** [review]
I see that there are still a few blocks and templates where we use a `isNewIdentity` variable based on `window.p4ge_vars.planet4_options.new_identity_styles`, shouldn't we remove those too? 🤔

File: assets/src/styles/blocks/Columns.scss [hunk 6]
```diff
@@ -256,10 +256,10 @@
           --block-columns--card-header-- {
             font-weight: 700;
             font-size: $font-size-xs;
-            font-family: var(--headings--font-family);
-            background: $dark-blue;
-            color: $white;
+            font-family: var(--font-family-heading);
           }
+          color: (--white);
```
Comment (mleray) [inline assets/src/styles/blocks/Columns.scss:261]: Isn't `var` missing here?

**Review by mleray — APPROVED** [review]
Code looks good to me 👍 Awesome job! 👏

**Pull request merged**